velox/exec/prefixsort/PrefixSortEncoder.h

```diff
@@ -53,6 +54,28 @@ class PrefixSortEncoder {
     }
   }
 
+  /// Encode String types.
+  FOLLY_ALWAYS_INLINE void encode(
+      std::optional<StringView> value,
+      char* dest,
+      int32_t encodeSize) const {
+    if (value.has_value()) {
+      dest[0] = nullsFirst_ ? 1 : 0;
+      std::string storage;
+      auto data = HashStringAllocator::contiguousString(value.value(), storage);
+      std::memcpy(dest + 1, data.data(), data.size());
```

Comment:
Why not just do `memcpy(dest + 1, value->data(), value->size())`

Reply:
`value` is retrieved from `row` by `value = *(reinterpret_cast<T*>(row + rowColumn.offset()));`. For string I think we need to read it by `contiguousString`. Please let me know if I missed some thing. Thanks.